backend/controller/dal/fsm_test.go

```diff
@@ -15,6 +16,8 @@ import (
 )
 
 func TestSendFSMEvent(t *testing.T) {
+	_ = observability.InitControllerObservability()
```

Comment:
I wasn't sure it was a good idea, but I think that having to do this everywhere is a good argument for statically initialising observability at `init()` time.

Reply:
Yeah totally agree. That would not be fun to have to remember this for every test that might have a metric. `init()` sounds good to me. Will probably clean up quite a bit of this plumbing too.